### PR TITLE
handle accountingInfo in import/1; add accountingInfo to cdrs

### PR DESCRIPTION
**Pull request by RefathWadood**
Handle accounting info from import file in cgf_tap module and include accounting info in CDRs https://github.com/sigscale/cgf/issues/1

### CODE REVIEW

File: src/cgf_tap.erl [hunk 1]
```diff
@@ -72,8 +72,11 @@ import(Filename, Log, Metadata)
 	end.
 %% @hidden
 import1(Filename, Log, Metadata, {ok, {transferBatch, TransferBatch}}) ->
-	#{callEventDetails := CDRs} = TransferBatch,
-	parse(Filename, Log, Metadata, CDRs);
+	#{callEventDetails := CDRs, accountingInfo := AccountingInfo} = TransferBatch,
```
Comment (vances) [inline src/cgf_tap.erl:75]: Pattern match in clause head.

File: src/cgf_tap.erl [hunk 2]
```diff
@@ -72,8 +72,11 @@ import(Filename, Log, Metadata)
...
+	parse(Filename, Log, Metadata, CDRs, AccountingInfo);
+import1(Filename, Log, Metadata, {ok, {transferBatch, TransferBatch}, _}) ->
+	#{callEventDetails := CDRs, accountingInfo := AccountingInfo} = TransferBatch,
```
Comment (vances) [inline src/cgf_tap.erl:78]: Pattern match in clause head.

File: src/cgf_tap.erl [hunk 3]
```diff
@@ -85,118 +88,130 @@ import1(Filename, _Log, _Metadata, {error, Reason}) ->
 %%  Internal functions
 %%----------------------------------------------------------------------
 
--spec parse(Filename, Log, Metadata, CDRs) -> Result
+-spec parse(Filename, Log, Metadata, CDRs, AccountingInfo) -> Result
```
Comment (vances) [inline src/cgf_tap.erl:91]: The list to be iterated over should be the last argument, the static arguments should be first.

File: src/cgf_tap.erl [hunk 4]
```diff
@@ -322,6 +337,174 @@ parse_message(_Log, _Metadata, _MessagingEvent) ->
...
+	when
+		AccountingInfo :: map(),
+		Result :: #{currencyConversionInfo => [map()],
+			discounting => map(),
+			localCurrency => list(),
+			tapCurrency => list(),
+			tapDecimalPlaces => integer(),
+			taxation => [map()]}| {error, Reason},
+		Reason :: term().
+%% @doc Parse CDRs from the import file.
```
Comment (vances) [inline src/cgf_tap.erl:350]: The description is wrong.

**Review by vances — COMMENTED** [review]
Style issues and an edoc error.

File: src/cgf_tap.erl (hunk 1 above)
Comment (RefathWadood) [reply src/cgf_tap.erl]: Fixed in [1fb53ef](https://github.com/sigscale/cgf/pull/11/commits/1fb53ef60f4a74d3b0ce77ae45bd92655659ddee)

File: src/cgf_tap.erl (hunk 2 above)
Comment (RefathWadood) [reply src/cgf_tap.erl]: Fixed in [1fb53ef](https://github.com/sigscale/cgf/pull/11/commits/1fb53ef60f4a74d3b0ce77ae45bd92655659ddee)

File: src/cgf_tap.erl (hunk 3 above)
Comment (RefathWadood) [reply src/cgf_tap.erl]: Fixed in [a79e7e6](https://github.com/sigscale/cgf/pull/11/commits/a79e7e63056759dbf9da154c28b660afe9592831)

File: src/cgf_tap.erl (hunk 4 above)
Comment (RefathWadood) [reply src/cgf_tap.erl]: Fixed in [4359ba1](https://github.com/sigscale/cgf/pull/11/commits/4359ba197ccbdcb5c95dd30ad3000ff55770bb21)

**Review by vances — APPROVED** [review]

**Pull request merged**